### PR TITLE
UTF8 encoding fix function

### PR DESCRIPTION
**Pull request by johncaling40**
Adds the UTF8 encoding fix function to prevent errors at tfrecord generation.

### CODE REVIEW

File: create_finetune_tfrecords.py [hunk 1]
```diff
@@ -240,8 +241,17 @@ def chunk_and_finalize(arrays, args, encoder):
     if not args.preserve_data_order:
         random.shuffle(full_seqs)
 
-    return full_seqs, trailing_data
+    return full_seqs, trailing_dat
```
Comment (siscia) [inline create_finetune_tfrecords.py:244]: is this a typo?

Comment (johncaling40) [reply create_finetune_tfrecords.py:244]: Yes

Comment (StellaAthena) [reply create_finetune_tfrecords.py:244]: @johncaling40 Can you fix it?

**Comment by kingoflolz** [comment]
Closed for inactivity

**Pull request closed without merging**